metrics/network/README.md

```diff
@@ -12,14 +12,17 @@ packet per second throughput, and latency.
 
 ## Networking tests
 
-- `network-metrics-iperf3.sh` measures bandwidth, jitter, bidirectional bandwidth,
+- `network-metrics-iperf3.sh` measures bandwidth, jitter,
```

Comment:
@GabyCT We are not using iperf3 because it does not handle bidirectional traffic at all?

Reply:
@amshinde yes exactly, `iperf3` does not have handle bidirectional

Reply:
@GabyCT ok. can you add it as a comment in the script so that this is documented.

Reply:
@amshinde , I put already a comment